crates/autopilot/src/domain/settlement/transaction.rs

```diff
@@ -12,7 +12,7 @@ pub struct Transaction {
     /// The block number of the block that contains the transaction.
     pub block: eth::BlockNo,
     /// The gas used by the transaction.
-    pub gas: eth::U256,
+    pub gas: eth::TokenAmount,
```

Comment:
not actually a token

Reply:
Why was this resolved? It's still marked as a token amount.

Reply:
@MartinquaXD I resolve comments as I do to keep track, and then I push it.

Reply:
> I resolve comments as I do locally to keep track, and then I push it.

Not really an intuitive approach from the reviewer's perspective 🙂

Reply:
> Not really an intuitive approach from the reviewer's perspective 🙂

What I can do is to put the PR in draft while reworking it. If that helps. But usually the time between me resolving a conversion and then pushing the code is not more than 10-15 minutes.